definitions/3.0.0/components.json

```diff
@@ -12,14 +12,7 @@
       "type": "object",
       "patternProperties": {
         "^[\\w\\d\\.\\-_]+$": {
-          "oneOf": [
-            {
-              "$ref": "http://asyncapi.com/definitions/3.0.0/Reference.json"
-            },
-            {
-              "$ref": "http://asyncapi.com/definitions/3.0.0/schema.json"
-            }
-          ]
+          "$ref": "http://asyncapi.com/definitions/3.0.0/anySchema.json"
```

Comment:
Dont remove reference, both has to be present, otherwise a schema component cannot be a reference.